packages/help-center/src/styles.scss

```diff
@@ -0,0 +1,116 @@
+.help-center__container {
+	background-color: #fff;
+	color: #000;
+
+	&.is-desktop {
+		background: white;
+		border-radius: 2px;
+		box-shadow: 0 0 3px 0 rgba( 0, 0, 0, 0.25 );
+		cursor: default;
+		display: inline;
+		min-width: 300px;
+		position: fixed;
+		right: 16px;
+		top: 74px;
+		width: 20vw;
+		z-index: 9999;
+		transition: 0.5s ease;
+
+		.help-center__container-header {
+			cursor: pointer;
+			height: 50px;
+		}
+
+		.help-center__container-content {
+			max-height: 70vh;
+			min-height: 300px;
+			overflow-y: auto;
+		}
+
+		&.is-minimized {
+			top: calc( 100vh - 65px );
+
+			.help-center__container-header {
+				cursor: default;
+			}
+		}
+	}
+
+	&.is-mobile {
+		position: fixed !important;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		background-color: #fff;
+		z-index: 9999;
+		transition: 0.5s ease;
+		max-height: 100vh;
+
+		&.is-minimized {
+			top: calc( 100vh - 50px );
+		}
+
+		.help-center__container-header {
+			height: 50px;
+		}
+
+		.help-center__container-content {
+			padding: 10px;
+			max-height: 90vh;
+			overflow-y: auto;
+		}
+	}
+
+	.components-button {
+		&:hover {
+			box-shadow: none;
+			color: inherit;
+		}
+	}
+
+	&.is-mobile.is-minimized,
+	&.is-desktop.is-minimized {
+		max-height: 50px;
+		transition: max-height ease 0.25s, top 0.5s ease 100ms;
+	}
+
+	@keyframes slideIn {
+		0% {
+			transform: translateX( 100vh );
+		}
+
+		100% {
+			transform: translateX( 0 );
+		}
+	}
+
+	@keyframes slideOut {
+		0% {
+			transform: translateX( 0 );
+		}
+
+		100% {
+			transform: translateX( 100vh );
+		}
+	}
+	@keyframes pullup {
+		0% {
+			transform: translateY( 100vh );
+		}
+
+		100% {
+			transform: translateY( 0 );
+		}
+	}
```

Comment:
I think we should only use `opacity` as shown in video [here](https://createp2.wordpress.com/2022/02/25/continuous-onboarding-guidance/). Movement is indicative and can mean something we don't want to convey.